### DISCUSSION TITLE
How to generate Fish completions in a program that uses urfave/cli?

### DISCUSSION THREAD

**Discussion by erazemk** [urfave/cli#1806] [Q&A] [answered]
A program I'd like to use uses `urfave/cli/v2`, but currently doesn't have fish completion, only bash. How can I generate completions for fish? `./<program> generate-completion fish` doesn't work, but that's the only example I found in the cli PRs, so I don't know what else to try.

**Comment by dearchap** [comment, marked as answer]
@erazemk You can try using App.ToFishCompletion()  https://github.com/urfave/cli/blob/v2-maint/fish.go#L13 and writing that to the fish complete file. Let me know if that doesnt work.

**Reply by erazemk** [reply]
That works, thanks!
